TombEngine/Game/items.cpp

```diff
@@ -461,20 +490,16 @@ void InitialiseItem(short itemNumber)
 	item->Animation.Velocity.y = 0;
 	item->Animation.Velocity.z = 0;
 
-	item->ItemFlags[3] = 0;
-	item->ItemFlags[2] = 0;
-	item->ItemFlags[1] = 0;
-	item->ItemFlags[0] = 0;
+	for (int i = 0; i < _countof(item->ItemFlags); i++)
```

Comment:
What is `_countof`? Do we have better way to count array size?

Reply:
hmm, i can add a constexpr with the size of ItemFlags which is 8 ?

Reply:
done